app/models/member.rb

```diff
@@ -5,10 +5,23 @@ class Member < ApplicationRecord
   mount_uploader :avatar, AvatarUploader
   validates :username, :firstname, :lastname, :email, :bio, presense: true
 
+  include AlgoliaSearch
```

Comment:
You might consider only including this _(and the block immediately below)_ if the application configuration option to use it is configured as `true`.